fix(TopBar): align lines

ShGKme opened the pull request:
### ☑️ Resolves

- With the default `line-height` one-line is 1.5 height of font
- In a limited spaces and with different font sizes, it looks a bit not aligned for me
  - The gap between lines is higher than between line and bottom border
  - Small text may look too small
- Using `normal` (about 1.2) line height ensures that the text is not cut off but also has no extra height even with normal font-size

## 🖌️ UI Checklist

### 🖼️ Screenshots / Screencasts

🏚️ Before | 🏡 After
-- | --
![image](https://github.com/user-attachments/assets/c47d9e5c-dd86-43d6-aa49-7885f08b75f3) | ![image](https://github.com/user-attachments/assets/d0d6d023-b99d-4775-bf31-55d19badb6e4)
![image](https://github.com/user-attachments/assets/9bd0ffb0-1a83-4c81-938f-150b33485627) | ![image](https://github.com/user-attachments/assets/255a242c-9fe7-4c38-99b3-b1891ba626a3)
![image](https://github.com/user-attachments/assets/fc103350-2cc3-48ad-8104-b7b6da59850b) | ![image](https://github.com/user-attachments/assets/45ca2225-9ac3-480f-87e2-44663ed6134f)

### 🏁 Checklist

- [x] 🌏 Tested with Chrome, Firefox and Safari or should not be risky to browser differences
- [x] 🖥️ Tested with Desktop client or should not be risky for it 
- [ ] 🖌️ Design was reviewed, approved or inspired by the design team
- [x] ⛑️ Tests are included or not possible
- [x] 📗 User documentation in https://github.com/nextcloud/documentation/tree/master/user_manual/talk has been updated or is not required

marcoambrosini commented:
I don't think this is ok for accessibility. Could check that?

ShGKme commented:
> I don't think this is ok for accessibility. Could check that?

With any solution it changes the top bar height, no matter what line height is set initially, if yuo mean this check.

![image](https://github.com/user-attachments/assets/225bb14e-2f2d-49e8-9074-89971f62e176)

ShGKme commented:
Updated to normal font-size with normal line-height everywhere

Antreesy reviewed (commented):
On Linux (evaluates to 22px):

B | A
-- | --
![image](https://github.com/user-attachments/assets/f2dfed38-c797-4851-93b5-1c649059b5e9) | ![image](https://github.com/user-attachments/assets/734c2d80-3a61-4182-beff-99f5b448a33b)

Diff:
![image](https://github.com/user-attachments/assets/4b6d1b14-fe3b-4113-8c96-df0e2bd8ba44)

ShGKme commented:
- Changed `normal` to `1.2` to make sure, it's the same in all browsers and OSs.
- Removed `--text-height` to make min-height based on similar elements size

Antreesy commented on src/components/TopBar/TopBar.vue:
```diff
@@ -303,15 +303,13 @@ export default {
 	align-items: center;
 	justify-content: flex-end;
-	min-height: calc(2 * var(--default-grid-baseline) + var(--text-height) + var(--border-width));
+	min-height: calc(var(--border-width) + 2 * (2 * var(--default-grid-baseline)) + var(--default-clickable-area));
```
Call Container uses magic number 60 or close, need to be updated to the same value there too

ShGKme replied:
Done in a hotfix

Antreesy replied:
Ok, but I thought, we're going for variables)

ShGKme replied:
I found it more complex and risky. `CallView` is used in different entrypoints and not directly related to `TopBar`

ShGKme commented:
Follow-up: use CSS variables in call view
